homeassistant/components/tibber/__init__.py

```diff
@@ -36,6 +36,12 @@
     import tibber
     tibber_connection = tibber.Tibber(conf[CONF_ACCESS_TOKEN],
                                       websession=async_get_clientsession(hass))
+
+    res = await tibber_connection.validate_login()
+    if res:
+        _LOGGER.error("Failed to setup Tibber. %s", res)
```

Comment:
it's weird that if a result exists, it means the login was invalid, since the method name is called `validate_login`.

We should probably update message:  "Failed to validate login"

Reply:
Maybe I should change the function name in pyTibber?
But I would like to return the error message, to be able to show it to the user.

Reply:
raise `InvalidLogin` exception ?